### PR TITLE
gst-plugins-good: patch to compile on < Lion

### PR DESCRIPTION
**Pull request by rhendric**
Patch accepted [upstream](http://cgit.freedesktop.org/gstreamer/gst-plugins-good/commit/?id=fc203a4bd7eb1cecc0e17bcb7ec67e0672806867).

### CODE REVIEW

File: Library/Formula/gst-plugins-good.rb [hunk 1]
```diff
@@ -50,6 +50,15 @@ class GstPluginsGood < Formula
 
   depends_on "libogg" if build.with? "flac"
 
+  if MacOS.version < :lion
+    # Snow Leopard and below don't have strnlen()
+    # https://bugzilla.gnome.org/show_bug.cgi?id=756154
+    patch do
+      url "http://cgit.freedesktop.org/gstreamer/gst-plugins-good/patch/?id=fc203a4bd7eb1cecc0e17bcb7ec67e0672806867"
```
Comment (DomT4) [inline Library/Formula/gst-plugins-good.rb]: Could you possibly submit this to https://github.com/Homebrew/patches and use that URL? We've found `cgit` URLs tend to churn the checksum every now and again and then we have issues verifying with upstream the change wasn't malicious.

Comment (DomT4) [inline Library/Formula/gst-plugins-good.rb]: If it's been accepted upstream you'll want to tuck this inside `stable do` further up as well.

Comment (rhendric) [inline Library/Formula/gst-plugins-good.rb]: It's also available from the [Bugzilla](https://bugzilla.gnome.org/attachment.cgi?id=313403&action=diff&context=patch&collapsed=&headers=1&format=raw); would Homebrew/patches still be preferable?

Comment (DomT4) [inline Library/Formula/gst-plugins-good.rb]: Bugzilla should be fine in this case then. That one shouldn't move around in terms of SHA.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/080d7efbd480fffc20003d088ee2f8744e5c38fb. Thank you for another contribution to Homebrew @rhendric! :smiley_cat:

**Pull request closed without merging**